lib/EnzymeCore/ext/AdaptExt.jl

```diff
@@ -1,11 +1,11 @@
 module AdaptExt
-    using Adapt
-    using EnzymeCore
+isdefined(Base, :get_extension) ? (using Adapt) : (using ..Adapt)
```

Comment:
If the extension is loaded unconditionally on older Julia versions, you can just use
```suggestion
using Adapt
```
(and not load Adapt in EnzymeCore itself). See #954.